build-final.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euo pipefail
+
+IMAGE=threema/webrtc-build-tools:latest
+TARGETS="${WEBRTC_TARGETS:-arm arm64 x86 x64}"
+BUILD_ARGS="${WEBRTC_BUILD_ARGS:-symbol_level=1 enable_libaom=false}"
```

Comment:
We now have the default environment variables in two places. Perhaps we could add one env file and use it for both?

Reply:
Fair point, but I'd just keep this for now, it's quite unlikely that this will change anytime soon. (And we have some duplication between this script and cli.sh anyways.)